lib/usecases/get_survey_details_use_case.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:injectable/injectable.dart';
+import 'package:lydiaryanfluttersurvey/api/repository/survey_repository.dart';
+import 'package:lydiaryanfluttersurvey/model/response/survey_detail_response.dart';
+
+import 'base/base_use_case.dart';
+
+@Injectable()
+class GetSurveyDetailsUseCase extends UseCase<SurveyDetailResponse, String> {
```

Comment:
Please update the test case name as well 🙏 
```suggestion
class GetSurveyDetailUseCase extends UseCase<SurveyDetailResponse, String> {
```